src/app/nav-menu/nav-menu.component.html

```diff
@@ -0,0 +1,5 @@
+<span id="hamburger-icon" appOcticon octicon="three-bars" width="30" [matMenuTriggerFor]="menu"></span>
+<mat-menu #menu="matMenu" [overlapTrigger]="false">
```

Comment:
I think our prototypes used a sidenav. If we want to use a regular menu we should be using the icon with the three vertical dots, not the hambuger, since that's the material design convention.

Reply:
Yeah, I'm fine with a sidenav if everyone thinks it's best. But depending on what we use this menu for, I feel like a full sidenav could be a bit overkill. It would be useful if we have lots of options and maybe some extra information to share in the menu, but if it's only going to list a handful of options for navigation, maybe the dropdown is best? What do you think? Any preferences?

Reply:
I think we might end up having about 7 menu items: Listings, Create Listing, My profile, My Offers, My listings, Settings, Log Out.